src/components/SingleName/SaveCancel.js

```diff
@@ -67,9 +73,15 @@ const SaveCancel = ({
   value,
   newValue,
   confirm,
+  oldcontentWarning,
   isValid = true
 }) => (
   <SaveCancelContainer className={className}>
+    {
```

Comment:
Same as above. Let's take a message prop which takes a string and a `showWarning` which is a boolean to know whether we need to show a warning or not

Reply:
Makes sense. I will rename to `warningMessage` so that I can pass the message there.